statviz/src/views/Statistics/StatisticsView.tsx

```diff
@@ -0,0 +1,45 @@
+import { gql, useQuery } from "@apollo/client";
+import { BeneficiaryDemographicsQuery, BeneficiaryDemographicsQueryVariables, HumanGender } from "../../types/generated/graphql";
+import DemographicChart, { IDemographicCube, IDemographicFact } from "./components/DemographicChart";
+
+const DEMOGRAPHIC_QUERY = gql`
+  query BeneficiaryDemographics {
+    beneficiaryDemographics {
+      facts {
+        count
+				createdOn
+				age
+				gender
+      }
+      dimensions {
+        tag {
+          __typename
```

Comment:
IN general, u do not have to add _typename. This is a default option of the InMemoryCache from Apollo